portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+public class Comment {
+  public long id;
+  public String comment;
+  public long timestamp;
+
+  public Comment(long id, String comment, long timestamp){
+    this.id=id;
+    this.comment=comment;
+    this.timestamp=timestamp;
+  }
+}
```

Comment:
tiny nit: if you add an extra newline after `}` this red icon will go away

Reply:
Addressed